### PR TITLE
Add runcam split support

### PR DESCRIPTION
**Pull request by azolyoung**
Add support for runcam split

### CODE REVIEW

File: tabs/auxiliary.js [hunk 1]
```diff
@@ -24,11 +24,41 @@ TABS.auxiliary.initialize = function (callback) {
...
+
+        return false;
+    }
+
+    function adjustRunCamSplitBoxNameWithModeID(modeId, originalModeName) {
+        switch (modeId) {
+            case 32: // BOXCAMERA1
+                return "CAMERA WI-FI";
+            case 33: // BOXCAMERA2
+                return "CAMERA POWERE";
```
Comment (McGiverGim) [inline tabs/auxiliary.js]: Little tipo: POWERE -> POWER

Comment (mikeller) [inline tabs/auxiliary.js]: Well spotted!

Comment (azolyoung) [inline tabs/auxiliary.js]: OK!

**Comment by azolyoung** [comment]
Hello
I've use rebase to combine all the commits to one, hope it will help for the review!

**Comment by azolyoung** [comment]
yes, It's need compatible with the old version firmware, i'm asking in the slack that i should use apiVersion or flightControllerVersion and the version number, I will fix this problem after the results.

**Comment by mikeller** [comment]
Simple: `apiVersion`.

**Comment by azolyoung** [comment]
OK, should i use condition: if apiVersion > 1.36.0 or other number??

**Comment by mikeller** [comment]
`apiVersion >= 1.36.0` is the correct check.

**Comment by azolyoung** [comment]
OK, thank you! i'v push a new commit.

**Comment by azolyoung** [comment]
@mikeller 
Hello, I've rebase my commits, and change the check logic for what firmwares are support runcam split, finally I use 1.37.0 to check, this is confirm from Dominic Clifton

**Comment by mikeller** [comment]
@azolyoung: As far as I can tell it should be 1.36 (next upcoming version), but maybe @hydra has other plans?

**Comment by azolyoung** [comment]
Yes, in the slack group, he said he will change the apiVersion to 1.37.0 before the firmware release

**Comment by mikeller** [comment]
That doesn't make sense, since that will break a lot of things. 1.36 is what the next release will be.

**Comment by azolyoung** [comment]
I think 1.37.0 is right, because 1.36.0 is using in the cf firmware that published, but these firmware(using 1.36.0) is no runcam split support, so if I use >= 1.36.0 to check whether the firmware support runcam split, then when the user using the configurator that support runcam split, and the cf firmware 2.0.4(the apiVersion is 1.36.0), It's show the runcam split support in ports, that is incorrect, because 2.0.4 is no runcam split.
Is this right?

**Comment by mikeller** [comment]
There is no released firmware with api version 1.36. But if you are ok with configurator not supporting runcam split for Betaflight 3.2, you can leave it at 1.37.

**Comment by azolyoung** [comment]
oh, sorry, i've check the apiVersion in firmwares 2.0.3 and 2.0.4, it's 1.35.0, yes, it's seems use 1.36.0 to check will be better then 1.37.0.
I'll ask Dominic about the detail

**Comment by azolyoung** [comment]
@mikeller 
Hello, following is the original words from Dominic, and in the cleanflight pull request, Dominic has submit a commit with the apiVersion change to 1.37 .0 
1.37.0.https://github.com/cleanflight/cleanflight/commit/3e6e303ce863fc7459ab5644983d12fabd7e0ae2

> hi @azol - there are a few boards in production with early firmware on, it will cause owners of those boards issues until they upgrade if we used 1.36.0.
[7:03] 
1.37.0 is best

**Comment by mikeller** [comment]
Will discuss this with @hydra, hang in there.

**Comment by azolyoung** [comment]
@mikeller 
OK, Thank you!

**Comment by azolyoung** [comment]
Is there any news about the condition?

**Comment by mikeller** [comment]
Sorry, have not heard back from @hydra yet, will try to chase him up.

**Comment by azolyoung** [comment]
Looking for you news! Thank you !

**Comment by hydra** [comment]
there is a bug in the code, the name replacement code doesn't work unless the serial configuration has been fetched from the FC first.  I will fix it.

**Comment by hydra** [comment]
@azolyoung see https://github.com/cleanflight/cleanflight-configurator/commit/ab1a0b26fd47a2acfa9548d5f9571b2f25d7e64a

merged.

**Comment by mikeller** [comment]
@hydra: MSP API version for Betaflight 3.2. is planned to be 1.36. Is it intentional that this will not be supported in Betaflight 3.2.0?

**Pull request merged**